ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -4298,7 +4343,7 @@ interface Port-Channel16
    isis authentication mode md5
    isis authentication key 0 <removed>
    spanning-tree guard none
-   switchport backup-link Port-Channel100.102 prefer vlan 20
+   switchport backup-link Port-Channel100 prefer vlan 20
```

Comment:
good catch - do we need to indicate this in the schema?

Reply:
yes, mentioned it